pkg/querier/codec/protobuf_codec.go

```diff
@@ -93,6 +93,7 @@ func createPrometheusQueryResponse(resp *v1.Response, cortexInternal bool) (*tri
 		ErrorType: string(resp.ErrorType),
 		Error:     resp.Error,
 		Warnings:  resp.Warnings,
+		Infos:     resp.Infos,
```

Comment:
Thanks for catching this. Can you please help add a unit test so that we don't run into the same issue again later?

Reply:
Test case added.